common/powershell/hyperv/hyperv.go

```diff
@@ -32,20 +32,27 @@ type scriptOptions struct {
 func GetHostAdapterIpAddressForSwitch(switchName string) (string, error) {
 	var script = `
 param([string]$switchName, [int]$addressIndex)
-
 $HostVMAdapter = Hyper-V\Get-VMNetworkAdapter -ManagementOS -SwitchName $switchName
 if ($HostVMAdapter){
-    $HostNetAdapter = Get-NetAdapter | ?{ $_.DeviceId -eq $HostVMAdapter.DeviceId }
-    if ($HostNetAdapter){
-        $HostNetAdapterIfIndex = @()
-        $HostNetAdapterIfIndex +=  $HostNetAdapter.ifIndex
-        $HostNetAdapterConfiguration =  @(get-wmiobject win32_networkadapterconfiguration -filter "IPEnabled = 'TRUE'") | Where-Object { $HostNetAdapterIfIndex.Contains($_.InterfaceIndex) }
-        if ($HostNetAdapterConfiguration){
-            return @($HostNetAdapterConfiguration.IpAddress)[$addressIndex]
-        }
-    }
+		$HostNetAdapter = Get-NetAdapter | Where-Object { $_.DeviceId -eq $HostVMAdapter.DeviceId }
+		if ($HostNetAdapter){
+				$HostNetAdapterIfIndex = @()
+				$HostNetAdapterIfIndex +=  $HostNetAdapter.ifIndex
+				$HostNetAdapterConfiguration =  @(get-wmiobject win32_networkadapterconfiguration -filter "IPEnabled = 'TRUE'") | Where-Object { $HostNetAdapterIfIndex.Contains($_.InterfaceIndex) }
+				if ($HostNetAdapterConfiguration){
+						return @($HostNetAdapterConfiguration.IpAddress)[$addressIndex]
+				}
+		}
+}
+else {
+	$HostNetAdapterConfiguration=@(Get-NetIPAddress -CimSession $env:computername -AddressFamily IPv4 | Where-Object { ( $_.InterfaceAlias -notmatch 'Loopback' ) -and ( $_.SuffixOrigin -notmatch "Link" )})
+	if ($HostNetAdapterConfiguration) {
+		 return @($HostNetAdapterConfiguration.IpAddress)[$addressIndex]
+	}
+	else {
+		return $false
+	}
```

Comment:
Not sure what's happened but could you fix the formatting here? The original code used an indent of four spaces rather than tabs

Reply:
Sure, don't know why my vscode formatted it this way.